README.md

```diff
@@ -1,138 +1,247 @@
+Trino client
+============
+A [Trino](https://trino.io/) client for the [Python](https://www.python.org/) programming language.
+
 [![Build Status](https://github.com/trinodb/trino-python-client/workflows/ci/badge.svg)](https://github.com/trinodb/trino-python-client/actions?query=workflow%3Aci+event%3Apush+branch%3Amaster)
 [![Trino Slack](https://img.shields.io/static/v1?logo=slack&logoColor=959DA5&label=Slack&labelColor=333a41&message=join%20conversation&color=3AC358)](https://trino.io/slack.html)
 [![Trino: The Definitive Guide book download](https://img.shields.io/badge/Trino%3A%20The%20Definitive%20Guide-download-brightgreen)](https://www.starburst.io/info/oreilly-trino-guide/)
 
-# Introduction
-
-This package provides a client interface to query [Trino](https://trino.io/)
-a distributed SQL engine. It supports Python>=3.6 and pypy.
-
-# Installation
+## Usage
+### The Python Database API (DBAPI)
 
+**Installation**
 ```
 $ pip install trino
 ```
 
-# Quick Start
+**Quick Start**
 
 Use the DBAPI interface to query Trino:
 
 ```python
-import trino
-conn = trino.dbapi.connect(
-    host='localhost',
-    port=8080,
-    user='the-user',
-    catalog='the-catalog',
-    schema='the-schema',
+from trino.dbapi import connect
+
+conn = connect(
+    host="<host>",
+    port=<port>,
+    user="<username>",
+    catalog="<catalog>",
+    schema="<schema>",
 )
 cur = conn.cursor()
-cur.execute('SELECT * FROM system.runtime.nodes')
+cur.execute("SELECT * FROM system.runtime.nodes")
 rows = cur.fetchall()
 ```
 
-This will query the `system.runtime.nodes` system tables that shows the nodes
-in the Trino cluster.
+This will query the `system.runtime.nodes` system tables that shows the nodes in the Trino cluster.
 
-The DBAPI implementation in `trino.dbapi` provides methods to retrieve fewer
-rows for example `Cursor.fetchone()` or `Cursor.fetchmany()`. By default
-`Cursor.fetchmany()` fetches one row. Please set
-`trino.dbapi.Cursor.arraysize` accordingly.
+The DBAPI implementation in `trino.dbapi` provides methods to retrieve fewer rows
+for example `Cursorfetchone()` or `Cursor.fetchmany()`. By default `Cursor.fetchmany()` fetches one row.
```

Comment:
`Cursor.fetchone()`

Reply:
fixed

Reply:
P/S: I just check the old docs and the code, I think `Cursor.fetchmany()` is correct.
https://github.com/trinodb/trino-python-client/blob/62f53c9d3ed9978e8a2bcd0eeacf2c7f6e071959/trino/dbapi.py#L481-L482